thirdparty/doctest/CHANGELOG.md

```diff
@@ -0,0 +1,410 @@
+# Change Log
```

Comment:
We don't need the CHANGELOG for a thirdparty library in our source.

Reply:
I'll remove this